src/modules/reading-room/queries/getReadingRooms.ts

```diff
@@ -0,0 +1,32 @@
+import ky from 'ky-universal';
+import * as queryString from 'query-string';
+import { useQuery } from 'react-query';
+import { UseQueryResult } from 'react-query/types/react/types';
+
+import { ApiResponseWrapper, ReadingRoomInfo } from '@reading-room/types';
+
+import { QUERY_KEYS } from 'constants/query-keys';
+
+async function getReadingRooms(searchInput?: string): Promise<ApiResponseWrapper<ReadingRoomInfo>> {
```

Comment:
Alle api calls samenzetten in een service, ik had er al eentje voorzien op deze branch: `reading-room/services/reading-room/reading-room.service`